web/ui/react-app/src/pages/targets/Filter.tsx

```diff
@@ -7,6 +7,10 @@ export interface FilterData {
   showUnhealthy: boolean;
 }
 
+interface Blah<T> {
```

Comment:
I'm not fully aware of this PR functionality but what is Blah?

Reply:
oops this was a mistake. I've removed it now.